### ISSUE TITLE
Table does not render the correct column names when group by tags

### ISSUE THREAD

**Issue by RobWin** [grafana/grafana#4839]
#* What grafana version are you using?
grafana-2.6.0-1

* What datasource are you using?
InfluxDB v0.12

* What OS are you running grafana on?
Suse Enterprise Linux 12

When I create a Table and create a Select statement which groups by tags, the column names do not match to the tag names. 
It seems that the date time picker is also not working correctly. I selected yesterday, but the table also shows the day before yesterday. Maybe I have configured the template interval wrong?

![image](https://cloud.githubusercontent.com/assets/5815046/14877716/331cfe3e-0d1f-11e6-9905-2054dcdad286.png)

Anything else I should provide?

**Comment by RobWin** [comment]
Generated request looks as follows: 

```
https://xxx/grafana/api/datasources/proxy/1/query?epoch=ms&q=SELECT+sum(%22value%22)+AS+%22Requests%22+FROM+%22requests%22+WHERE+%22hostname%22+%3D+%27spica01.aul.t-online.de%27+AND+time+%3E+1461708000s+and+time+%3C+1461794400s+GROUP+BY+time(1d),+%22application%22,+%22appId%22,+%22className%22,+%22methodName%22
````

**Comment by torkelo** [comment]
try Grafana 3.0, 

think this is the same issue reported in https://github.com/grafana/grafana/issues/3928 , fixed in master / 3.0

**Comment by RobWin** [comment]
ok. Thx :)
